### DISCUSSION TITLE
Namespace of Schema not found when compiling.

### DISCUSSION THREAD

**Discussion by DerTomDerTwitch** [jamescourtney/FlatSharp#379] [Q&A]
I added a fbs scheme to my project and all the code is generated properly so far. Auto-complete works fine, but when I want to build my solution, the namespace is not found and I can't build an executable. Any ideas what might cause this or how I can get more information on this error?

**Comment by jamescourtney** [comment]
Hi @JefferiesTube --

Can you share a sample project that exhibits the behavior? I'd need a bit more to go off of to drill into this. Are you using .NET? Unity? Something else?

Thanks!
